wasm-engines.md

```diff
@@ -0,0 +1,224 @@
+# Wasm Engines
+
+Binaryen and Wabt are two WebAssembly Engines which also provides a set of tools
+for compiling and decompiling wasm binary files. Both are official WebAssembly Projects.
+
+# Binaryen
+
+## Getting and compiling binaryen
+
+Install development tools (`build-essential`) `cmake` and `make`:
+
+```
+sudo apt-get install build-essential cmake make
+```
+
+Clone the official Binaryen repository:
+
+```
+git clone https://github.com/WebAssembly/binaryen.git
+```
+
+Move to the new `binaryen` directory and run this command to build the tools:
```

Comment:
Consider adding a link to the [official build instructions](https://github.com/WebAssembly/binaryen#building) for reference, and in case they change